src/Idris/ProcessIdr.idr

```diff
@@ -235,7 +235,7 @@ processMod : {auto c : Ref Ctxt Defs} ->
              (sourcecode : String) ->
              Core (Maybe (List Error))
 processMod srcf ttcf msg sourcecode
-    = catch (do
+    = assert_total $ catch (do -- TODO: Check totality
```

Comment:
I added this due to a change in totality checking after despecializing in the concurrent-idris branch. Will investigate if this assert can somehow be removed.

Reply:
Couldn't figure out a way to remove the assert :/